src/test/java/io/appium/java_client/pagefactory_tests/widget/tests/combined/CombinedWidgetTest.java

```diff
@@ -161,4 +165,33 @@ public List<PartiallyCombinedWidget> getWidgets() {
             return multipleWidgets;
         }
     }
+
+
+    /**
+     * Assert proxy class cache growth for this test class.
+     * The (@link io.appium.java_client.proxy.Helpers#CACHED_PROXY_CLASSES) should be populated during these tests.
+     * Prior to the Caching issue being resolved
+     * - the CACHED_PROXY_CLASSES would grow indefinitely, resulting in an Out Of Memory exception.
+     * - this ParameterizedTest would have the CACHED_PROXY_CLASSES grow to 266 entries.
+     */
+    private void assertProxyClassCacheGrowth() {
+        System.gc(); //Trying to force a collection for more accurate check numbers
+        int thresholdSize = 50;
```

Comment:
maybe it would make more sense to move this value into a class constant and add a comment to it